config/util_nix.go

```diff
@@ -0,0 +1,11 @@
+// +build !windows
+
+package config
+
+import "syscall"
+
+func umask() int {
+	umask := syscall.Umask(022)
```

Comment:
Can you restore the comment about how `syscall.Umask` sets the umask (so we have to unset it, which we do on L9)?

Reply:
Sure.